### ISSUE TITLE
multiple quadcopters?

### ISSUE THREAD

**Issue by remzinho** [3demax/elfie#2]
Is there a way to support connectivity to multiple quads?
Can the IP of each drone be modified somehow?

**Comment by 3demax** [comment]
My drone's wifi is in AP mode, so if you have 2-3 drones, you should also have 2-3 wlan interfaces for RPi.
I doubt it's possible to change IP unless you change the firmware of a drone somehow.

Are you looking to build some kind of swarm of drones?

**Comment by remzinho** [comment]
@3demax yes. i did something similar with some parrot minidrones node libs, but never with the parrot AR since it was actually a similar AP and you'd have to connect to it.
